core/cap-0038.md

```diff
@@ -255,17 +250,20 @@ index 0e7bc842b..2976045dc 100644
      } trustLine;
  
  case OFFER:
-@@ -479,6 +550,9 @@ case CLAIMABLE_BALANCE:
+@@ -479,6 +545,12 @@ case CLAIMABLE_BALANCE:
      {
          ClaimableBalanceID balanceID;
      } claimableBalance;
 +
 +case LIQUIDITY_POOL:
-+    PoolID liquidityPoolID;
++    struct
++    {
++        PoolID liquidityPoolID;
++    } liquidityPool;
```

Comment:
why a struct here with a single field?

Reply:
This follows the convention followed by `claimableBalance`, which allows the key to use https://github.com/stellar/stellar-core/blob/f31c8f90d7abc634fc89818e013a32dc5f2badc8/src/bucket/LedgerCmp.h#L54. We could use a single field here and work around that method instead, but I thought it made sense to follow what we'd done previously.